frontend/src/actions/dataset.js

```diff
@@ -220,6 +220,25 @@ export function fetchDatasets(skip = 0, limit = 21, mine = false) {
 	};
 }
 
+export const RECEIVE_MY_DATASETS = "RECEIVE_MY_DATASETS";
+
+export function fetchMyDatasets(skip = 0, limit = 21, mine = true) {
+	return (dispatch) => {
+		// TODO: Parameters for dates? paging?
+		return V2.DatasetsService.getDatasetsApiV2DatasetsGet(skip, limit, true)
```

Comment:
Any reason you can reuse the fetch Datasets? I see you are hardcoding "mine" to be true in the function. Wondering if it's the same effect as pass in mine=true as parameter

Reply:
I had added myDatasets since I added a new field in the root state, so it has a new method that sets it. 
I can try to just use the same state though, looking at it now I think that it should work fine with just 1 state.